modules/op-release-notes-1-2.adoc

```diff
@@ -13,6 +13,8 @@
 * Tekton `tkn` CLI 0.13.1
 * Tekton Triggers 0.8.1
 * ClusterTasks based on Tekton Catalog 0.16
+* IBM Z and LinuxONE on RHOCP 4.6
+* IBM Power Systems on RHOCP 4.6
```

Comment:
@Preeticp We should probably mention that user need to install 1.2 from `preview` channel (and remove that note later when https://issues.redhat.com/browse/SRVKP-988 is resolved).

While it's confusing for x86 architecture (but installation from `ocp-4.6`  works, you only get older release), the installation from `ocp-4.6` channel on P&Z will fail because 1.1 doesn't exist on those architectures.

Reply:
@SNiemann15,  Pavol and Vincent are right, you will need to add a note for both power and Z. Also, please start watching the Jira so that you know when the Jira is resolved and you can update the note.

Reply:
Today we talked about promoting 1.2.1 into `ocp-4.6` channel so it maybe this note will not be necessary. Still waiting for approval

Reply:
Instead of RHOCP I would use {product-title}

Right now, we don't commonly use RHOCP in our doc set, though that might change. There were some recent discussions about that. For now, I would avoid its usage.